teuthology/task/mpi.py

```diff
@@ -1,12 +1,12 @@
 """
 Start mpi processes (and allow commands to be run inside process)
 """
-from StringIO import StringIO
 import logging
 import re
 
 from teuthology import misc as teuthology
 from six import string_types as basestring
+from six import StringIO
```

Comment:
The String is mentioned only once in this file, could you drop it at all?
Just replace this:
```
version_str = remote.run(args=["mpiexec", "--version"], stdout=StringIO()).stdout.getvalue()
```
with
```
version_str = remote.sh("mpiexec --version")
```

Reply:
@kshtsk i fixed a couple of regressions introduced by s/run/sh/ . and i'd like to avoid doing this again if possible. we can always revisit these `run()` calls and refactor them after the py3 migration. but at this moment, i'd prefer having a minimum change which only addresses the py3 incompatibility.

Reply:
I guess it was that typo issues, same can appear with switching `six.StringIO` stuff.

Reply:
@tchaikov still, could you please make an exclusion for me here, there is no a reason to use StringIO here? thanks

Reply:
okay, will do.

Reply:
> I guess it was that typo issues, same can appear with switching `six.StringIO` stuff.

i disagree.

Reply:
updated and repushed.